PageObjectPattern/src/test/resources/features/apitest/spotifyPostPlayList.feature

```diff
@@ -0,0 +1,16 @@
+#access token is generated manually
+Feature: Sending POST requests
```

Comment:
What about "Sending POST requests" -> "Integration tests for v1/playlist endpoint"?